src/containers/App/App.test.js

```diff
@@ -72,7 +72,7 @@ describe('App', () => {
 
     it('should render the NoteForm container when a valid note is searched', () => {
 
-      const mockNotes = [{ id: 1, title: "Title One" }, { id: 2, title: "Title Two" }]
+      const mockNotes = [{ id: '1', title: "Title One" }, { id: '2', title: "Title Two" }]
```

Comment:
Good catch here. I know we switched from Date.now() to ShortID so it makes sense that these are coming in as strings.